server/config/const.go

```diff
@@ -4,13 +4,13 @@
 package config
 
 const (
-	BotUserName    = "msoffice"
-	BotDisplayName = "Microsoft Office TODO"
-	BotDescription = "Created by the Microsoft Office Plugin. TODO"
+	BotUserName    = "mscalendar"
+	BotDisplayName = "Microsoft Calendar Bot"
+	BotDescription = "Created by the Microsoft Calendar Plugin."
 
-	ApplicationName = "Microsoft Office"
-	Repository      = "mattermost-plugin-msoffice"
-	CommandTrigger  = "msoffice"
+	ApplicationName = "Microsoft Calendar"
+	Repository      = "mattermost-plugin-mscalendar"
+	CommandTrigger  = "mscalendar"
```

Comment:
bot and command display name changes

Reply:
Suggest `mscal` for command? @aaronrothschild

Reply:
(can be a separate change though, not blocking for this PR)